docs/source/markdown/options/userns.container.md

```diff
@@ -4,16 +4,16 @@
 ####> are applicable to all of those.
 #### **--userns**=*mode*
 
-Set the user namespace mode for the container. It defaults to the **PODMAN_USERNS** environment variable. An empty value ("") means user namespaces are disabled unless an explicit mapping is set with the **--uidmap** and **--gidmap** options.
+Set the user namespace mode for the container. It defaults to the **PODMAN_USERNS** environment variable. An empty value ("") or **host** means user namespaces are disabled, unless an explicit mapping is set with the **--uidmap** and **--gidmap** options or podman is rootless.
 
 This option is incompatible with **--gidmap**, **--uidmap**, **--subuidname** and **--subgidname**.
 
-Rootless user --userns=Key mappings:
+User --userns=Key mappings:
 
-Key       | Host User |  Container User
-----------|---------------|---------------------
-""        |$UID         |0 (Default User account mapped to root user in container.)
-keep-id   |$UID         |$UID (Map user account to same UID within container.)
+Key       | Host User   |  Container User
+----------|-------------|---------------------
+"" or host|$UID         | 0 (Default User account mapped to root user in container.)
+keep-id   |$UID         | $UID (Map user account to same UID within container.)
```

Comment:
Does this work in rootful mode.
# podman run --userns=keep-id alpine echo hi
Error: keep-id is only supported in rootless mode